src/xopen/__init__.py

```diff
@@ -1061,7 +1062,7 @@ def xopen(
 
 @overload
 def xopen(
```

Comment:
This breaks backward-compatibility for people who have filename as a keyword argument.

Reply:
Whoops, it was not my intention to sneak in something unrelated like this. Fixed now.